decidim-conferences/config/locales/en.yml

```
@@ -288,7 +288,23 @@ en:
           error: There was a problem updating a registration type for this conference.
           success: Conference registration type successfully updated.
       titles:
+        conference_admins: Conference admins
+        conference_invites: Conference invites
+        conference_partners: Conference partners
+        conference_registrations: Conference registrations
+        conference_speakers: Conference speakers
         conferences: Conferences
+        duplicate_conference: Duplicate - %{title} - conference
+        media_links: Media links
+        new_conference: New conference
+        new_conference_admin: New conference admin
+        new_conference_invite: New conference invite
+        new_conference_partner: New conference partner
+        new_conference_speaker: New conference speaker
+        new_media_link: New media link
+        new_registration_type: New registration type
+        registration_types: Registration types
+        update_conference_speaker: Update conference speaker
```

For consistency with other keys: 
```suggestion
        edit_conference_speaker: Update conference speaker
```
I'd leave the "Update ..." as it is what we're already using in the view title.